tests/test_schema_syntax.py

```diff
@@ -1521,6 +1530,37 @@ def test_eschema_syntax_alias_03(self):
         };
         """
 
+    @tb.must_fail(errors.EdgeQLSyntaxError,
+                  r"Missing ',' before the shape item 'baz'", line=5, col=17)
+    def test_eschema_syntax_alias_04(self):
+        """
+        module test {
+            alias FooBaz := Foo {
+                val := 1
+                baz := .bar + 2
+            }
+            );
+        };
+        """
+
+    @tb.must_fail(errors.EdgeQLSyntaxError,
+                  r"Missing ',' before the tuple item '2'", line=3, col=32)
+    def test_eschema_syntax_alias_05(self):
+        """
+        module test {
+            alias FooBaz := (1 2);
```

Comment:
Are you sure it's a tuple and not just a fat-fingered parenthesized expr?

Reply:
Good point. We should rewrite error messages to use "if", as in "hey, if this meant to be a tuple you'd want to have a comma between 1 and 2", or something along these lines.

Reply:
I've reworded the error messages to be in the form of "It appears that ..." since that is still accurate. Now the message states what the parser seems to infer (based on Nonterminals that's what the parser was trying to match). So whatever the human intent might have been, but the actual text appears to be <blah>. This phrasing should be helpful either way: if the intent matches, then the hint is relevant, if the intent is surprising, then the error indicates what the parser sees and that can be helpful in figuring out what went wrong in that chunk of code, too.